Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -240,6 +240,7 @@ class AccessibilityObject : public AXCoreObject, public CanMakeWeakPtr<Accessibi
     bool hasSameStyle(const AXCoreObject&) const override { return false; }
     bool hasUnderline() const override { return false; }
     bool hasHighlighting() const override;
+    std::optional<CharacterRange> textInputMarkedRange() const override;
```

Comment:
Can/should this be annotated with `final`?

Reply:
Yeah, @twilco said that we should be doing that for these types.